lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -212,7 +212,7 @@ defmodule Bamboo.SMTPAdapter do
   end
 
   defp format_email({nil, email}), do: email
-  defp format_email({name, email}), do: "<#{email}> #{name}"
+  defp format_email({name, email}), do: "#{name}<#{email}>"
   defp format_email(emails) when is_list(emails) do
```

Comment:
It's possible if the space is required by the spec. Do you know what the spec says on the official format?